driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/ODBFeatures.java

```diff
@@ -107,7 +108,7 @@ public boolean supportsComputer() {
 
         @Override
         public boolean supportsTransactions() {
-            return false;
+            return transactionalGraph;
```

Comment:
I agree with @velo - the tinkerpop3 doc suggests that you can ask the graph up front if it supports a feature or not, independent if we enabled transaction on this instance or not:
`graph.features().graph().supportsTransactions()`

http://tinkerpop.incubator.apache.org/docs/3.0.2-incubating/

As I don't have push access to your branch I'll merge this to master and change it there. Let me know if I'm missing anything.